### ISSUE TITLE
Build from source did not have vminsert and vmselect on Mac M1 cpu laptop

### ISSUE THREAD

**Issue by renchili** [VictoriaMetrics/VictoriaMetrics#1576]
**Describe the bug**
followed this [build](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#production-builds) guid run the command `make` on this architecture: 
```
Darwin LIs-MacBook-Pro.local 20.5.0 Darwin Kernel Version 20.5.0: Sat May  8 05:10:31 PDT 2021; root:xnu-7195.121.3~9/RELEASE_ARM64_T8101 arm64
```
the bin folder is like this :
![image](https://user-images.githubusercontent.com/33677006/131277449-6a05592f-10aa-40ee-9c4a-9e350810f631.png)

**To Reproduce**
follow the guid 

**Expected behavior**
have vminsert and vmselect built

**Logs**
NA

**git related version**:
commit 4b3877b798c29881290eba1dd467f92436bbe151

**Comment by renchili** [comment]
for more in the Main Makefile it wont do anything about vminsert and vmselect
![image](https://user-images.githubusercontent.com/33677006/131277829-bbc13397-0789-4c7e-938c-95afbd4c5d51.png)
But your guid said it will built binary to bin folder:
![image](https://user-images.githubusercontent.com/33677006/131277864-1dffc97d-2e0e-4ebd-b36f-60b31dc87aed.png)

**Comment by denisgolius** [comment]
Hi @renchili !
You have builded VictoriaMetrics Single that contains 
 vminsert functionality. Thats why it have name VM Single. To get vminsert as a separate binary file you should switch to cluster branch https://github.com/VictoriaMetrics/VictoriaMetrics/tree/cluster and run build againe. See this https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#building-from-sources.  To get information about cluster scheme - check this link https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#architecture-overview.

**Comment by renchili** [comment]
but i needs  to test this issue https://github.com/VictoriaMetrics/VictoriaMetrics/issues/1559
i did not see this has been merged to cluster branch from any commit and change
what should i do then?

**Comment by valyala** [comment]
The change has been included in cluster branch at 7fdb4db73d2b4e2b17d84e5e8f1f2e0f2bc06555 . Note that the change is relevant only to a single-node VictoriaMetrics and to `vmagent`, which is usually built from single-node branch. So it is recommended building `vmagent` from `master` branch, while building all the cluster components - vminsert, vmselect and vmstorage - from `cluster` branch.

**Comment by renchili** [comment]
thanks your reply let me try
